apertium_apy/apy.py

```diff
@@ -178,11 +183,13 @@ def apply_config(args, parser, apy_section):
 
 
 def parse_args(cli_args=sys.argv[1:]):
+    langNames = 'langNames.db'
     parser = argparse.ArgumentParser(description='Apertium APY -- API server for machine translation and language analysis')
     parser.add_argument('pairs_path', help='path to Apertium installed pairs (all modes files in this path are included)')
     parser.add_argument('-s', '--nonpairs-path', help='path to Apertium tree (only non-translator debug modes are included from this path)')
     parser.add_argument('-l', '--lang-names',
-                        help='path to localised language names sqlite database (default = langNames.db)', default='langNames.db')
+                        help='path to localised language names sqlite database (default = ' + langNames + ')',
```

Comment:
Use string formatting not concatenation.

Reply:
Done